sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultPodSpec.java

```diff
@@ -65,7 +67,8 @@ public DefaultPodSpec(
             @JsonProperty("volumes") Collection<VolumeSpec> volumes,
             @JsonProperty("pre-reserved-role") String preReservedRole,
             @JsonProperty("secrets") Collection<SecretSpec> secrets,
-            @JsonProperty("share-pid-namespace") Boolean sharePidNamespace) {
+            @JsonProperty("share-pid-namespace") Boolean sharePidNamespace,
+            @JsonProperty("allow-decommission") Boolean allowDecommission) {
```

Comment:
**Out of scope of this PR:** but it seems as if this is duplicated in `RawPod`.

Reply:
`Raw*` are the direct YAML schema, while this layer is what's used internally. `YAMLToInternalMappers` handles translating from the prior to the latter